docs/03-policies/prodev.md

```diff
@@ -0,0 +1,102 @@
+
+Also, if you’re asked by someone else to assist with their prodev, please make time to help them out!
+
+## Communication
+
+We have a [#prodev](https://civicactions.slack.com/messages/prodev) Slack channel for prodev-related communication.
+
+## Prodev FAQ's
+
+Q: How much do I get to spend on prodev each year?  
+A: Your budget is $1,200. If you’re just starting, you’ll need to wait 90 days before you gain access to your prodev budget. If you’re a part-time employee, your budget may be less than $1,200. Your manager can tell you more.
+
+Q: What kinds of stuff can I spend my prodev budget on?  
+A: Books, courses, conferences, certifications and prodev-related travel expenses.
+
+Q: Are there any other limits on the prodev budget?  
+A: For spending amounts over $50, you’ll need to get approval in advance. [Use this form](https://goo.gl/forms/4z1GQsBtrx6m4lVy2) to request approval. Anything under $50 can be purchased without permission as long as it supports professional development goals. Receipts and expense sheets should be submitted in the usual way, noting as professional development costs.
+
+Q: Do I have to spend my prodev money during a scheduled prodev sprint?  
+A: Nope. Spend it any time you like. 
+
+Q: How can I find out how much of my yearly prodev budget has been used?  
+A: See the summary tab of this sheet:  [Prodev Usage 2018](https://docs.google.com/spreadsheets/d/1fi1B2Gx37NpBtZxY8yR_J0hfKRxGzMy4vSn764IxTRA/edit#gid=98617197)
+
+Q: What if I want to spend more money than I have budget for?  
+A: Talk to your manager if you have prodev needs that can’t be covered by your annual budget. 
+
+Q: Does my prodev budget carry over to the following year if I don’t use it all?  
+A: No. Prodev budgets are “use it or lose it”, so go get busy learning something!
+
+Q: Do I have to use my prodev hours allocation during a scheduled prodev sprint?  
+A: Nope. Spend it any way you like. But remember that focusing your efforts during a prodev sprint can make the time spent much more effective.
+
+
+## Appendix and Other Info
+
+### Ways to expand/improve prodev efforts:
+
+* How do we measure prodev progress? What would be some prodev KPI’s?
+    * Graph: hours spent for prodev, per person
+    * Money invested in prodev, maybe per person?  Monthly?
+    * Participation numbers: what percentage of the company used at least 1 hour of prodev time? This could be tracked and reported on each quarter, with a goal of 100% participation.
+    * Would also be helpful to find historical participation percentage if the data exists to calculate it
+    * Monthly "state of prodev" report
+* How do we set prodev goals on both an individual and company-wide level?
```

Comment:
I am floating the idea of service area leads helping with VSI for their respective area. Maybe prodev goals can be split up by area?